salt/orch/update.sls

```diff
@@ -410,7 +433,7 @@ remove-caasp-fqdn-grain:
 
 masters-remove-update-grain:
   salt.function:
-    - tgt: G@roles:kube-master and G@update_in_progress:true
+    - tgt: '{{ is_updateable_master_tgt }}'
```

Comment:
This one is a little different: `is_updateable_master_tgt` expands to `G@tx_update_reboot_needed:true and * and G@roles:kube-master` whereas this was `G@roles:kube-master and G@update_in_progress:true`. `update_in_progress:true` vs `tx_update_reboot_needed: true`.

Reply:
Good catch. I'll fix it. Thanks.

Reply:
Done.